docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -92,6 +92,14 @@ <h3>[property:Boolean autoClearStencil]</h3>
 			Default is *true*.
 		</p>
 
+		<h3>[property:Boolean programCheckEnabled]</h3>
+		<p>
+			If [page:.programCheckEnabled programCheckEnabled] is true, defines whether Material shader programs are checked
+			for errors during compilation process. It may be useful to disable this check in production for performance gain.
+			It is strongly recommended to keep these checks enabled during development.
```

Comment:
Maybe it's worth mentioning what happens if the shader doesn't compile? The material just won't render, right?

Reply:
done